docs/metrics.md

```diff
@@ -130,7 +130,11 @@ Following metrics record stats around activation handling within Controller
 Aggregate metrics for inflight activations.
 
 * `openwhisk.histogram.loadbalancer<controllerId>_activationsInflight_count` (histogram) - Records the number of activations being worked upon for a given controller. As a histogram it would give a distribution of inflight activation count within a flush interval.
-* `openwhisk.histogram.loadbalancer<controllerId>_memoryInflight_count` (histogram) - Records the amount of RAM memory in use for in flight activations. This is not actual runtime memory but the memory specified per action limits.
+* `openwhisk.histogram.loadbalancer<controllerId>_memory<actionType>Inflight_count` (histogram) - Records the amount of RAM memory in use for in flight activations. This is not actual runtime memory but the memory specified per action limits. actionType defines whether it is a managed or a blackbox action
+
+Metrics below are for current memory capacity
+
+* `openwhisk.histogram.loadbalance_totalCapacity<invokerType>_count` (histogram) - Current memory capacity for all usable managed and blackbox invokers, total user memory in shard managed by controller. invokerType defines whether it is a managed or a blackbox invoker.
```

Comment:
Typo in metric name: `loadbalance`.

Reply:
I understand that this is just documenting the current code so the question comes too late... why is the metric not per `<controllerId>`?

Reply:
@sven-lange-last - Thanks for the comment. The current grafana display, does work on the basis of different controller. The query in grafana is so designed that we take in result from all the controllers(2 or more) and display them as a total. @chetanmeh I would be interested to know that will this metrics in its current form would also be useful for your current usage of metrics.